tests/scripts/build-release.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -ex
+
+#############
+# FUNCTIONS #
+#############
+
+function  build() {
+    # set VERSION to a dummy value since Jenkins normally sets it for us. Do this to make Helm happy and not fail with "Error: Invalid Semantic Version"
+    build/run make VERSION=0 build.all
+    build/run make mod.check
+}
+
+if echo "${BRANCH_NAME}" | grep "release-" ; then
```

Comment:
The steps for master and release branches should really be the same. The only difference I can think of is that a non-tagged release branch would *not* run the promote action (for the helm chart). So we should be able to combine these steps, and just have a condition only on the promote.

Reply:
we have `TAG_WITH_SUFFIX=true` for release so I think we can't combine anymore.

for `release`
`build/run make -C build/release build BRANCH_NAME=${BRANCH_NAME} TAG_WITH_SUFFIX=true GIT_API_TOKEN=${GIT_PSW}`

for `master`

` build/run make -C build/release build BRANCH_NAME=${BRANCH_NAME}  GIT_API_TOKEN=${GIT_PSW}`

Reply:
Got it, I overlooked the TAG_WITH_SUFFIX